### ISSUE TITLE
Cython is installed but doesn't seem to do anything

### ISSUE THREAD

**Issue by ForceBru** [holzschu/Carnets#148]
I can successfully execute `%%load_ext cython` and then run a cell like this:

```
%%cython -a --verbose

def test():
    return 5 + 6
```

This outputs:

```
[1/1] Cythonizing /var/mobile/Containers/Data/Application/<huge UUID>/Documents/.ipython/cython/_cython_magic_92a929e066e52d7b4b226cb8d367b669af63f47f.pyx
building '_cython_magic_92a929e066e52d7b4b226cb8d367b669af63f47f' extension
```

...and then it completes without any errors, but calling `test()` keeps raising `NameError`.

I assume this is because there's no C compiler, like in https://github.com/holzschu/Carnets/issues/125, but why have Cython if there's no C compiler? Am I missing something?

---------

Carnets version: 1.5.5

**Comment by holzschu** [comment]
You are right in your assumption: Cython is limited because there are no C compilers. Even if there was a C compiler, loading dynamic libraries is heavily restricted under iOS.

It is installed mostly because it is requireed by other extensions, so these other extensions install it as part of their dependencies.

**Comment by ForceBru** [comment]
Oh, this is sad - I was hoping for some Cython action(
